core/src/main/java/com/oath/oak/OakRValueBufferImpl.java

```diff
@@ -13,12 +13,16 @@
 
 public class OakRValueBufferImpl implements OakRBuffer {
```

Comment:
Maybe it would be better to have two OakRBuffer implementations: fixed-reference and non-fixed reference. It was good to have an impl with a fixed reference.

Reply:
The difference between two classes would be to small... I prefer to leave it like this, if OakRBuffer needs to be immutable just don't change it's buffer...